tensorboard/uploader/uploader_test.py

```diff
@@ -355,6 +356,65 @@ def test_upload_skip_large_blob(self):
         self.assertEqual(0, mock_rate_limiter.tick.call_count)
         self.assertEqual(1, mock_blob_rate_limiter.tick.call_count)
 
+    def test_filter_graphs(self):
+        # Three graphs: one short, one long, one corrupt.
+        bytes_0 = _create_example_graph_bytes(123)
+        bytes_1 = _create_example_graph_bytes(9999)
+        bytes_2 = b"\x0a\x7fbogus"  # invalid (truncated) proto
```

Comment:
Please add a comment about what these values mean and why they are necessary for the test.  `"bogus"` alone was sufficient to trigger `DecodeError`.  Is this the case that sometimes produces `RuntimeWarning`?

Reply:
That's all good; I'm just saying that you can't expect a reader to recognize and decode the proto wire format on sight.  A comment would solve that:
```
# Declare a proto field of length 127 bytes (`\x7f`), but provide only 5 bytes.
```
Similarly explaining the \x0a would be nice.

Reply:
Okay, added an explicit comment per your suggestion.